style.css

```diff
@@ -0,0 +1,74 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+.wrapper {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  align-content: space-around;
+}
+
+.mylogo {
+  color: #6070ff;
+  font-family: Poppins;
```

Comment:
- [ ] Your linter problem is because you are missing to add a secondary generic font family fallback in case that your Poppins one doesn't work. Please try something like ``` font-family: Poppins, sans-serif;``` and check if it works :) [support article](https://stylelint.io/user-guide/rules/list/font-family-no-missing-generic-family-keyword/)